README.md

```diff
@@ -11,6 +11,20 @@ alike, without causing conflicts with any other dependency.
 NOTE:  Although the binary artifact produced by the project is backwards-compatible with Java 8, you do need 
        JDK 9 or higher to modify or build the source code of this library itself.
 
+This Change
```

Comment:
👍 
This is a great explanation, but it should probably be moved below in this `README.md`
Maybe as a sub-section of [Key Value Secret Engine Config](https://github.com/BetterCloud/vault-java-driver/blob/master/README.md#key-value-secret-engine-config)

Reply:
Definitely agree with this.  The placement of this text here is pretty bizarre, and future people reading the README will have no context of which PR/change this paragraph is talking about.

I'll re-work this myself, post-merge.

Reply:
Yes, I intended for you to move it to release notes once a release number had been assigned.